config/metasrv.example.toml

```diff
@@ -58,17 +58,32 @@ max_metadata_value_size = "1500KiB"
 
 # Failure detectors options.
 [failure_detector]
+
+## The threshold value used by the failure detector to determine failure conditions.
```

Comment:
What's the unit of this number, second? Or it's just a number in PHI

Reply:
Yes, it's just a number...